.github/workflows/frontend-ci.yml

```diff
@@ -78,6 +78,7 @@ jobs:
       DJANGO_DANDI_WEB_APP_URL: http://localhost:8085
       DJANGO_DANDI_API_URL: http://localhost:8000
       DJANGO_DANDI_JUPYTERHUB_URL: https://hub.dandiarchive.org/
+      DJANGO_DANDI_INSTANCE_NAME: DANDI
```

Comment:
Is there a reason why only `DJANGO_DANDI_INSTANCE_NAME` appears here? Is it because the other values are not tested in CI?

It might be good to include all the variables here so that the setup is complete (even if some aspects of the setup are not being tested or relied on).